pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -21,10 +23,12 @@ import (
 const (
 	// DIDExchange did exchange protocol
 	DIDExchange        = "didexchange"
-	connectionSpec     = metadata.AriesCommunityDID + ";spec/connections/1.0/"
+	connectionSpec     = metadata.AriesCommunityDID + ";spec/didexchange/1.0/"
 	connectionInvite   = connectionSpec + "invitation"
 	connectionRequest  = connectionSpec + "request"
 	connectionResponse = connectionSpec + "response"
+	//TODO : : Acknowledgement needs to follow RFCS-0015 for acks : https://github.com/hyperledger/aries-rfcs/tree/master/features/0015-acks
+	connectionAck = connectionSpec + "ack"
```

Comment:
[type](https://github.com/hyperledger/aries-rfcs/tree/master/features/0015-acks#explicit-acks) of ack message is did:sov:BzCbsNYhMrjHiqZDTUASHg;spec/notification/1.0/ack

Reply:
@rolsonquadras not in this case; consider `ack` as being [adopted](https://github.com/hyperledger/aries-rfcs/blob/master/concepts/0003-protocols/template.md#adopted-messages) by the `did-exchange` protocol